pkg/reconciler/wasmmodule/wasmmodule_test.go

```
@@ -0,0 +1,85 @@
+		t.Fatalf("read golden file: %v", err)
+	}
+
+	var gotMap, wantMap any
+	if err := json.Unmarshal([]byte(got), &gotMap); err != nil {
+		t.Fatalf("unmarshal got: %v", err)
+	}
+	if err := json.Unmarshal(goldenBytes, &wantMap); err != nil {
+		t.Fatalf("unmarshal golden: %v", err)
+	}
+
+	gotJSON, _ := json.MarshalIndent(gotMap, "", "  ")
```

golangci-lint (2.5.0)

[error] Error return value of `encoding/json.MarshalIndent` is not checked: unsafe type `any` found

(errchkjson)